src/tests/unit/connection.test.js

```diff
@@ -0,0 +1,79 @@
+import chai from 'chai';
+import chaiExclude from 'chai-exclude';
+import chaiAsPromised from 'chai-as-promised';
```

Comment:
I believe chaiAsPromised should come before chaiExclude

Reply:
Looks like I don't even use it for this test so I can just remove it anyway.